cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,55 @@
+import faker from 'faker';
+
+import PurchaseFormPageObject from '../support/pages/purchaseForm.cy';
+import ClickOnTheLaptops from '../support/pages/laptopPage.cy';
+import MainPage from '../support/pages/visitWebSite.cy';
+import Cart from '../support/pages/cartPage.cy';
+
+const purchaseForm = new PurchaseFormPageObject();
+const laptopPage = new ClickOnTheLaptops();
+const website = new MainPage();
+const checkIn = new Cart();
 /// <reference types='cypress' />
 
-describe('', () => {
-  before(() => {
+const testingData = {
+  name: faker.name.firstName(),
+  countryname: faker.address.country(),
+  cityname: faker.address.city(),
+  card: '9999 9999 9999 9999',
+  month: faker.date.month(),
+  year: '2023'
+};
 
+describe('make order on the website', () => {
+  beforeEach(() => {
+    website.visit();
   });
 
-  it('', () => {
+  it('should select laptop and add to the cart', () => {
```

Comment:
what is the difference between this test and the next? remove it, this is useful because there is no assertion here

Reply:
I was thinking of doing 3 **independent**  tests. That's why I separate the flow.